src/main.cpp

```diff
@@ -108,6 +109,7 @@ int main(int argc, char *argv[])
         "-unload %s",    &unloadFiles,   "Remote: unload loaded files matching the given (unix shell style) pattern",
         "-quit",         &quitRemote,    "Remote: close the existing displaz window",
         "-add",          &addFiles,      "Remote: add files to currently open set, instead of replacing those with duplicate labels",
+        "-modify",       &mutateData,    "Remote: mutate data already loaded with the matching label (requires displaz .ply with an \"index\" field to indicate mutated points)",
```

Comment:
So usage is

```
displaz -rmtemp -modify -label mydataset /tmp/blah.ply
```

Seems reasonable, though perhaps not entirely clear usage.  Would `-modifylabel` make more sense as an alternative to `-label`:

```
displaz -rmtemp -modifylabel mydataset /tmp/blah.ply
```

?